cw_bitcoin/lib/litecoin_wallet.dart

```diff
@@ -908,7 +959,30 @@ abstract class LitecoinWalletBase extends ElectrumWallet with Store {
     // https://github.com/ltcmweb/mwebd?tab=readme-ov-file#fee-estimation
     final preOutputSum =
         outputs.fold<BigInt>(BigInt.zero, (acc, output) => acc + output.toOutput.amount);
-    final fee = utxos.sumOfUtxosValue() - preOutputSum;
+    var fee = utxos.sumOfUtxosValue() - preOutputSum;
+
+    // determines if the fee is correct:
+    BigInt _sumOutputAmounts(List<TxOutput> outputs) {
+      BigInt sum = BigInt.zero;
+      for (final e in outputs) {
+        sum += e.amount;
+      }
+      return sum;
+    }
+
+    print("fee is initially: $fee");
+
+    final sum1 = _sumOutputAmounts(outputs.map((e) => e.toOutput).toList()) + fee;
+    final sum2 = utxos.sumOfUtxosValue();
+    if (sum1 != sum2) {
+      print("@@@@@ WE HAD TO ADJUST THE FEE! @@@@@@@@");
+      final diff = sum2 - sum1;
+      // add the difference to the fee (abs value):
+      fee += diff.abs();
+    }
+
+    print("fee is now: $fee");
```

Comment:
this was mostly a sanity check since we were getting that sum value of utxo error and the only way that is thrown is if these values are different
what's also weird to me is that most of the time the fee appears to be 0 (I think for mweb-mweb tx's)

Reply:
were you able to reproduce the error anyway?

Reply:
no I haven't been able to reproduce the issue since the initial fix we had for silent payments